pkg/config/environment_containers.go

```diff
@@ -62,6 +62,18 @@ func init() {
 	registerFeature(Podman)
 }
 
+// IsAnyContainerFeaturePresent checks if any of known container features is present
+func IsAnyContainerFeaturePresent() bool {
+	return IsFeaturePresent(Docker) ||
+		IsFeaturePresent(Containerd) ||
+		IsFeaturePresent(Cri) ||
+		IsFeaturePresent(Kubernetes) ||
+		IsFeaturePresent(ECSFargate) ||
+		IsFeaturePresent(EKSFargate) ||
+		IsFeaturePresent(CloudFoundry) ||
+		IsFeaturePresent(Podman)
+}
```

Comment:
architectural nit: I know it's not directly needed for this PR but this whole file is ripe for turning it into a feature-detection Provider-pattern module